code/__DEFINES/movespeed_modification.dm

```diff
@@ -82,6 +82,17 @@
 
 #define MOVESPEED_ID_SLOW_ARMOR		                    	"SLOW_ARMOR "
 
+#define MOVESPEED_ID_SLOW_SUITBOX_CAP	               	"SLOW_SUITBOX_CAP"
```

Comment:
why not just use one define for all suitboxes? I feel like having one for each is just needless clutter.

Reply:
because when you pick up two boxes and drop one, you'll be freed from the movement slow while you're still holding one.

Reply:
hmm, actually, I got an idea how to make it single define...

Reply:
it's quite better now. I checked it's working.